src/engine/enginebuffer.h

```diff
@@ -389,6 +393,7 @@ class EngineBuffer : public EngineObject {
     QAtomicInt m_iEnableSyncQueued;
     QAtomicInt m_iSyncModeQueued;
     ControlValueAtomic<QueuedSeek> m_queuedSeek;
+    bool m_previousBufferSeek = false;
```

Comment:
If this is accessed from multiple threads it should be an atomic for proper memory barriers.

Reply:
It isn't, this value is only read/written in the processSeek function